dashboard/app/models/section_hidden_script.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: section_hidden_scripts
+#
+#  id         :integer          not null, primary key
+#  section_id :integer          not null
+#  script_id  :integer          not null
+#
+# Indexes
+#
+#  index_section_hidden_scripts_on_script_id   (script_id)
+#  index_section_hidden_scripts_on_section_id  (section_id)
+#
+
+class SectionHiddenScript < ApplicationRecord
+  belongs_to :section
+  belongs_to :script
+end
```

Comment:
Please keep the migration to its own PR, deployed separate from any references or usages of the model.

Reply:
It is...

Reply:
Migration was https://github.com/code-dot-org/code-dot-org/pull/17261

Reply:
Ah, so this PR just needs rebasing. Never mind, ignore me...